README.md

```diff
@@ -64,7 +64,7 @@ var s3 = new AWSLIB.AWS.S3({
 
 The AWS SDK can be customized for specific API versions and/or services.
 
-This project defaults to the following services: `ses,s3,lambda`.
+This project defaults to the following services: `ses,s3,lambda,ec2`.
```

Comment:
In this case `.npmrc` needs to be modified to add ec2 to the default list of services.